tools/data-handler/src/show.ts

```diff
@@ -281,9 +281,14 @@ export class Show {
      * @todo: add unit tests
      * @returns all templates in a project.
      */
-    public async showTemplatesWithDetails(projectPath: string): Promise<(resource)[]> {
+    public async showTemplatesWithDetails(projectPath: string): Promise<template[]> {
         Show.project = new Project(projectPath);
-        return await Show.project.templates();
+        const promiseContainer = [];
```

Comment:
Suggestion: could it be cleaner to use map over templates instead a for loop and push?

Reply:
Yes it would, but the same syntax is also used elsewhere. I replace the for loop with map in this one